src/sentry/utils/snuba.py

```diff
@@ -115,14 +131,16 @@ def query(start, end, groupby, conditions=None, filter_keys=None,
 
     assert expected_cols == got_cols
 
-    for d in response['data']:
-        if 'time' in d:
-            d['time'] = int(to_timestamp(parse_datetime(d['time'])))
-        for col in rev_snuba_map:
-            if col in d:
-                d[col] = rev_snuba_map[col][d[col]]
+    with timer('process_result'):
+        for d in response['data']:
+            if 'time' in d:
+                d['time'] = int(to_timestamp(parse_datetime(d['time'])))
+            for col in rev_snuba_map:
+                if col in d:
+                    d[col] = rev_snuba_map[col][d[col]]
+        result = nest_groups(response['data'], groupby, aggregate_cols)
 
-    return nest_groups(response['data'], groupby, aggregate_cols)
+    return result
```

Comment:
You can just return out of the context manager instead of assigning to the local on L141 if you'd like.